Source/WebCore/platform/graphics/MediaPlayerPrivate.cpp

```diff
@@ -51,6 +51,7 @@ const PlatformTimeRanges& MediaPlayerPrivateInterface::seekable() const
 {
     if (maxTimeSeekable() == MediaTime::zeroTime())
```

Comment:
I would tend to do: `auto maxTimeSeekable = this->maxTimeSeekable()` given this is a virtual method.